cloudlaunchserver/templates/cl-celery-deployment.yaml

```diff
@@ -58,11 +58,9 @@ spec:
               protocol: TCP
           env:
           {{- include "cloudlaunchserver.envvars" . }}
-{{- range $key, $val := .Values.extra_env }}
-{{- if $val }}
-            - name: {{ $key | upper }}
-              value: "{{ tpl $val $ }}"
-{{- end }}
+{{- if .Values.extra_env }}
+{{- $nowhitespace := (regexReplaceAll "{{\\s*([^}\\n]+)\\n?([^}\\n]+)?\\n?([^}\\n]+)?\\n?([^}\\n]+)?\\n?([^}\\n]+)?\\s*}}" (toYaml .Values.extra_env) "{{$1 $2 $3 $4 $5}}") -}}
```

Comment:
Think it would be better if this long regex was moved to a template instead of being repeated.